dxp/frontend_taglib_clay.md

```diff
@@ -0,0 +1,49 @@
+# What is `frontend-taglib-clay` and it's relation with clay-\*, @clayui, and css?
+
+## High-level:
+
+`frontend-taglib-clay` provides a collection of clay dependencies from npm and custom components to be used across DXP as taglibs and js dependencies. Versions of these npm modules are locked in this module, this is done to keep modules consistent across dxp. For example, by locking these versions, this means we use the same version of `@clayui/button` across all of dxp so that we can reduce the number of modules served with dxp.
+
+## How is it structured:
```

Comment:
If this is a question, end with "?" and not ":"

(Otherwise, reword as a non-question;  ie. "How it is structured".)